sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -299,7 +299,8 @@ def post_port_sfp_info_to_db(logical_port_name, table, transceiver_dict,
                      ('application_advertisement', port_info_dict['application_advertisement']
                       if 'application_advertisement' in port_info_dict else 'N/A'),
                      ('is_replaceable', str(is_replaceable)),
-                     ('dom_capability',port_info_dict['dom_capability']),
+                     ('dom_capability', port_info_dict['dom_capability']
+                      if 'dom_capability' in port_info_dict else 'N/A'),
```

Comment:
Do you think we should add this check for all the fields and raise a seperate PR for that ?
It will save us from future regressions

Reply:
That's tricky, because it's a slippery slope, as it would allow new platforms to *not* provide support for a field which all other platforms currently support. I think we should only apply it to new fields until we have refactored sonic-platform-common and all vendors have migrated to the shared parser implementations.